### PR TITLE
Added Ukrainian localization

### PR DESCRIPTION
**Pull request by foresle**
I tried to add Ukrainian localization and used this documentation - https://developer.android.com/guide/topics/resources/localization
I hope I didn't miss anything.

### CODE REVIEW

**Comment by sfan5** [comment]
The CI build is failing, your file should be at ` app/src/main/res/values-uk/strings.xml`.
Also please remove all entires with `translatable="false"`.

File: app/src/main/res/values-uk/strings.xml [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+    <!-- Simple strings. -->
+    <string name="mpv_activity">mpv</string>
+    <string name="title_activity_settings">Налаштування</string>
+    <string name="dialog_ok">OK</string>
+    <string name="dialog_cancel">Скасувати</string>
+    <string name="dialog_save">Зберегти</string>
+    <string name="dialog_yes">Так</string>
+    <string name="dialog_no">Ні</string>
+    <string name="dialog_prev">Попереднє</string>
+    <string name="dialog_next">Наступне</string>
```
Comment (sfan5) [inline app/src/main/res/values-uk/strings.xml:14]: grafik
These might be too long for the button, is there an abbreviation that can be used here?

File: app/src/main/res/values-uk/strings.xml [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+    <string name="error_no_file">Не вказано жодного файлу, допобачення :D</string>
+    <string name="notice_file_appended">Файл додано до плей-листа</string>
+    <string name="exit_warning_playlist">%d елементи залишились в списку відтворення.\nВи точно хочете вийти?</string>
+    <string name="contrast">Контрасність</string>
+    <string name="video_brightness">Яскравість відео</string>
+    <string name="gamma">Гамма</string>
+    <string name="saturation">Насиченість</string>
+    <string name="aspect_ratio">Співвідношення сторін</string>
+    <string name="audio_delay">Затримка звуку</string>
+    <string name="sub_delay">Затримка ce,nbnhsd</string>
```
Comment (sfan5) [inline app/src/main/res/values-uk/strings.xml:41]: Looks like a mistake

File: app/src/main/res/values-uk/strings.xml (hunk 1 above)
Comment (foresle) [reply app/src/main/res/values-uk/strings.xml]: Yes

File: app/src/main/res/values-uk/strings.xml (hunk 2 above)
Comment (foresle) [reply app/src/main/res/values-uk/strings.xml]: Yes

**Comment by sfan5** [comment]
Merged as 1f6216b0a3b12fe9b576e9d39bd468aafac710ee, thanks.

**Pull request closed without merging**